Bugzilla/Bug.pm

```diff
@@ -4411,6 +4412,60 @@ sub bug_alias_to_id {
     undef, $alias);
 }
 
+# Last changed timestamp of a non-bot user
+sub delta_ts_non_bot {
+  my ($self) = @_;
+  return $self->{delta_ts_non_bot} if exists $self->{delat_ts_non_bot};
```

Comment:
Typo: `$self->{delat_ts_non_bot}` -> `$self->{delta_ts_non_bot}`

I suppose this means this is missing test coverage.

Reply:
Nah just means that the cached value is never used so technically would still work but be less performant. Sorry for missing the misspelling.